src/main/java/org/folio/circulation/resources/ItemsInTransitResource.java

```diff
@@ -1,28 +1,53 @@
 package org.folio.circulation.resources;
 
+import static java.util.concurrent.CompletableFuture.completedFuture;
```

Comment:
This file appears to be using a different import ordering to the [provisionally agreed standard](https://issues.folio.org/browse/FOLIO-1667). 

Given that this change is introducing lots of new imports, please can it be made to fit with this ordering.

Reply:
What if we include this order imports to the build check? Because it is hard to follow such chances.

Reply:
Yes, It would be nice to include static analysis as a build check for this. We haven't done that as the policy has been wholly adopted and we haven't had the bandwidth to integrate the tooling to do this yet.

For the time being, we are trying to spread the knowledge for folks to try to use a consistent ordering.